lib/Service.php

```diff
@@ -189,6 +189,27 @@ function write($rootElementName, $value, $contextUri = null) {
 
     }
 
+    /**
+     * Maps the given plain-old-php-object (popo) to the given rootElementName within the given namespace.
```

Comment:
Ok, so how should the method be named?

`mapVo` or `mapValueObject`? I prefer the latter